src/org/zaproxy/zap/utils/HirshbergMatcher.java

```diff
@@ -41,15 +41,6 @@
     // Maximum value for comparison ratio
     public static final double MAX_RATIO = 1.0;
     
-    /**
-     * Algorithm B as described by Hirschberg
```

Comment:
Is this and the similar C comment below worth leaving, even as a single line comment for future maintenance,  or do you think the function name is enough?